packages/@aws-cdk/toolkit/lib/api/io/private/codes.ts

```diff
@@ -0,0 +1,54 @@
+import { ToolkitError } from '../../errors';
+import { IoMessageCode } from '../io-message';
+
+export const CODES = {
```

Comment:
some of these codes could be consolidated and/or their numbers could make a little more sense, but i personally have no idea if there should be any rhyme or reason to the codes. so leaving it as is in this PR